[{
	"raw": {},
	"required": [
		"Filename",
		"Size"
	],
	"package": "python3-lib389",
	"source": "389-ds-base",
	"version": "2.0.15-1ubuntu2",
	"section": "universe/net",
	"priority": "optional",
	"architecture": "all",
	"depends": [
		"python3-argcomplete",
		"python3-argparse-manpage",
		"python3-dateutil",
		"python3-distro",
		"python3-ldap",
		"python3-pkg-resources",
		"python3-pyasn1",
		"python3-pyasn1-modules",
		"python3:any",
		"libnss3-tools",
		"openssl",
		"python3-packaging",
		"python3-pytest"
	],
	"replaces": [
		"389-ds-base (<< 1.4.0.18-1~)",
		"python-lib389 (<< 1.3.7.8)"
	],
	"conflicts": [
		"389-ds-base (<< 1.4.0.18-1~)",
		"python-lib389 (<< 1.3.7.8)"
	],
	"installedSize": 2145,
	"maintainer": "Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>",
	"description": "Python3 module for accessing and configuring the 389 Directory Server",
	"homepage": "https://directory.fedoraproject.org",
	"filename": "pool/universe/3/389-ds-base/python3-lib389_2.0.15-1ubuntu2_all.deb",
	"size": 366244,
	"md5": "ed963d1f51d0086310ca3d6c6cae8855",
	"sha1": "d773686c6fafccb1714b6ba828edbb1497d7a009",
	"sha256": "5cb5fda63cfde63edf4e2cb44a785612fe425fa79eca1b013a4e13a20908e875",
	"sha512": "5ab51b8e51706dfaef79efda30eb911fe90312d6f9f0a89f3028749cf88db488a8f6e2668350743035ee2049afe039372a489ad60856c9e590d7acce0344f419",
	"descriptionMd5": "312a10ddcf41c03aed17c8e2759b4410"
},{
	"raw": {},
	"required": [
		"Filename",
		"Size"
	],
	"package": "python3-lib389",
	"source": "389-ds-base",
	"version": "2.0.15-1",
	"section": "universe/net",
	"priority": "optional",
	"architecture": "all",
	"depends": [
		"python3-argcomplete",
		"python3-argparse-manpage",
		"python3-dateutil",
		"python3-distro",
		"python3-ldap",
		"python3-pkg-resources",
		"python3-pyasn1",
		"python3-pyasn1-modules",
		"python3:any",
		"libnss3-tools",
		"openssl",
		"python3-packaging",
		"python3-pytest"
	],
	"replaces": [
		"389-ds-base (<< 1.4.0.18-1~)",
		"python-lib389 (<< 1.3.7.8)"
	],
	"conflicts": [
		"389-ds-base (<< 1.4.0.18-1~)",
		"python-lib389 (<< 1.3.7.8)"
	],
	"installedSize": 2145,
	"maintainer": "Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>",
	"description": "Python3 module for accessing and configuring the 389 Directory Server",
	"homepage": "https://directory.fedoraproject.org",
	"filename": "pool/universe/3/389-ds-base/python3-lib389_2.0.15-1_all.deb",
	"size": 366186,
	"md5": "4920d62019f73f7e2bf19e567caeafba",
	"sha1": "6679f5795d2d23f855013b82a26bdd8fb6ea6f7e",
	"sha256": "7d9178369676be71a8238411a5c91fbd761906909ef4b387ecd48771c1f9a660",
	"sha512": "be4060e74d56d2a0a406eacb069597cbafbdf2dc6b6267429ed49cbd9dd4d19b214014dcf0eb1aec68a9992fe15cb0a01f68d18f3e558a7f83d5fb7bc6a446b9",
	"descriptionMd5": "312a10ddcf41c03aed17c8e2759b4410"
}]